spec/instrumentation/command_subscriber_spec.rb

```diff
@@ -64,8 +64,11 @@
       expect(tracer.spans.last.finished?).to be true
 
       # there should be an error tag and message log
-      expect(tracer.spans.last.tags["error"]).to be true
-      expect(tracer.spans.last.logs.count).to be 1
+      span = tracer.spans.last
+      puts span.tags
```

Comment:
not a big deal but looks like sth you added when debugging and haven't removed it